### ISSUE TITLE
Improve entry point documentation to SDK generation from API spec

### ISSUE THREAD

**Issue by maririos** [Azure/azure-sdk-tools#9383]
Now that our users are using Typespec, we need to guide them on documentation to configure the emitters in their projects. 
The current entry point is: https://github.com/Azure/azure-rest-api-specs/blob/main/documentation/Getting-started-with-TypeSpec-specifications.md#arm which doesn't explain too much.

Things to add:
- How to configure the emitters
- Which ones
- Who can help?
- Why is it needed

Should we move this documentation to EngHub too?

**Comment by raych1** [comment]
This is the entry page for `TypeSpec` in EngHub site, which includes more information about other areas. 

This one is more about how to work with `TypeSpec` in spec repo. I would suggest to add reference to this document for other knowledges, such as how to write SDK emitter configurations.

**Comment by maririos** [comment]
> This one is more about how to work with TypeSpec in spec repo. I would suggest to add reference to this document for other knowledges, such as how to write SDK emitter configurations.

You mean doing a different document specific to this, right? if so yes I agree

**Comment by raych1** [comment]
Yes, I recommend having a different page specific to the SDK emitter configurations.
